tests/test_block/test_create_block.py

```diff
@@ -104,3 +105,23 @@ def test_delete_non_existent_block(self):
             delete_non_existent_block,
             delete_non_existent_block_response
         )
+
+    def test_delete_block_different_location(self):
```

Comment:
Use more descriptive names for your tests to reflect what is being tested for. E.g `test_delete_block_in_different_location_throws_error`

Reply:
@Bonifase, I've updated the names. Some were too long and didn't leave room for adding `throws_errors`.